M2/Macaulay2/packages/RationalMaps.m2

```diff
@@ -1238,7 +1238,7 @@ beginDocumentation();
 
 document {
     Key => RationalMaps,
-    Headline => "rational maps",
+    Headline => "A package for computations with rational maps.",
```

Comment:
"A package for computations with " is redundant.

And remove the period.

Reply:
Changed to
rational maps between varieties